test/io/jade/readBits-resultType.chpl

```diff
@@ -26,4 +26,4 @@ assert(t3 == 0b010);
 r.close();
 f.close();
 
-// FileSystem.remove(testfile);
+FileSystem.remove(testfile);
```

Comment:
@jabraham17 - just checking to make sure, was this intentionally left commented out or accidentally?

Reply:
That was left commented out accidentally, thanks for fixing it!